data_team/metabase_server/docker-compose.yml

```
@@ -48,8 +37,8 @@ services:
       - MB_REDIRECT_ALL_REQUESTS_TO_HTTPS=false
       - MB_SITE_URL=null
       - MB_DB_DBNAME=metabase
-      - MB_DB_HOST=postgres
+      - MB_DB_HOST=<get the public IP from GCP Cloud SQL's db instance>
```

How about taking env vars from `.env`? May need to modify the instruction in the readme and add a new key in `.env.example` (and delete the old one) as well.
```suggestion
      - MB_DB_HOST=${CLOUD_SQL_URI}  # rename it
```

---

@david30907d would you take care of this one? or you'd like to remove the `.env`?

---

oh sorry for the late reply, will do it tmrw~

---

done~